installers/rpm-code/dev-env.sh

```diff
@@ -26,3 +26,7 @@ export install_proprietary_key=proprietary_code
 # for convenience to test rpm-perl, not used here
 export rpm_perl_install_dir=$(_root)/radiasoft/rsconf/rpm
 unset _root
+if [[ ! -e $radiasoft_repo_file ]]; then
```

Comment:
Seems like this should be embedded in dev-setup.sh so that the sentinel is modular.

Reply:
There is recursion with dev-env.sh now so this won't work anyway. 

I think this code could use more functions and possibly be a single file, e.g. dev-env.sh, which does all the work. It could take an argument "server". If the server isn't running, it would tell the user to run dev-env.sh server.